app/models/commit_monitor_branch.rb

```diff
@@ -20,6 +20,11 @@ def self.github_commit_uri(user, repo, sha = "$commit")
     "https://github.com/#{user}/#{repo}/commit/#{sha}"
   end
 
+  def enabled_for?(checker)
```

Comment:
:+1: i had the same the same thought but we stopped ourselves here. i'd like to write some tests for this worker (though that could be a bigger investment of time), so didn't want to refactor too much until that was done.

@carbonin do you have bandwidth to look at this today?